src/models/stores/exemplar-controller.ts

```diff
@@ -184,17 +213,16 @@ export const ExemplarControllerModel = BaseExemplarControllerModel
     }
   }))
   .actions(self => ({
-    async initialize(user: UserModelType, db: DB) {
-      const hide = db.stores.appConfig.initiallyHideExemplars;
+    async initialize(stores: IStores) {
```

Comment:
In store implementations that need to access other stores, a pattern I've tried to follow is for them to define their own IStores like interface which just includes the stores that the implementation needs. This way it makes it easier to document what stores depend on what other stores, and it sometimes eliminates a circular dependency.

You can still pass the full stores object to the initializer, it just changes the declaration here.